holidays/countries/united_states.py

```
@@ -378,475 +391,504 @@ def _populate_subdiv_holidays(self):
         super()._populate_subdiv_holidays()
 
     def _populate_subdiv_ak_public_holidays(self):
-        # Washington's Birthday
         if self._year >= 1971:
-            self._add_holiday_3rd_mon_of_feb("Presidents' Day")
+            # Presidents' Day.
+            self._add_holiday_3rd_mon_of_feb(tr("Presidents' Day"))
 
-        # Seward's Day
         # No observance in 1921: https://web.archive.org/web/20230208015211/https://chroniclingamerica.loc.gov/lccn/sn86072239/1922-03-29/ed-1/seq-8/
```

Pylint (3.3.7)

[convention] Line too long (149/100)

(C0301)